card/mod/05_standard/set/all/rich_html/menu.rb

```diff
@@ -2,7 +2,7 @@
   view :menu, :denial=>:blank, :tags=>:unknown_ok do |args|
     return _render_template_closer if args[:menu_hack] == :template_closer
     return '' if card.unknown?
-    wrap_with :div, :class=>'menu-slot' do
+    wrap_with :div, :class=>'menu-slot nodblclick' do
```

Comment:
Isn't that a good thing?  That would be selecting the first menu item, no?  Seems like what you would want that action to do.